docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,808 @@
+network over Bluetooth LE and then be commissioned into a Matter network.
+
+After connecting the device over Bluetooth LE, the controller will print the
+following log:
+
+```
+Secure Session to Device Established
+```
+
+This log will mean that the PASE (Password-Authenticated Session Establishment)
+session using SPAKE2+ protocol has been established.
+
+##### Commissioning into Thread network over Bluetooth LE
+
+To commission the device to the existing Thread network, use the following
+command pattern:
+
+```
+$ chip-tool pairing ble-thread <node_id> hex:<operational_dataset> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<operational_dataset\>_ is the Operational Dataset determined in the
+    step 3.
+-   _<pin_code\>_ and _<discriminator\>_ are device-specific keys determined in
+    the step 4.
+
+##### Commissioning into Wi-Fi network over Bluetooth LE
+
+To commission the device to the existing Wi-Fi network, use the following
+command pattern:
+
+```
+$ chip-tool pairing ble-wifi <node_id> <ssid> <password> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<ssid\>_ and _<password\>_ are credentials determined in the step 3.
+-   _<pin_code\>_ and _<discriminator\>_ are device-specific keys determined in
+    the step 4.
+
+If you prefer the hexadecimal format, use the `hex:` prefix. For example:
+
+```
+$ chip-tool pairing ble-wifi <node_id> hex:<ssid> hex:<password> <pin_code> <discriminator>
+
+```
+
+> **Note:** The _<node_id>_ can be provided as a hexadecimal value with the `0x`
+> prefix.
+
+#### Commissioning into a network over IP
+
+This option is available when the Matter device is already present in an IP
+network (Thread or Wi-Fi enabled), but it has not been commissioned to a Matter
+network yet.
+
+##### Commissioning with setup PIN code
+
+Use the following command pattern to discover devices and try to pair with the
+first discovered one using the provided setup code:
+
+```
+$ chip-tool pairing onnetwork <node_id> <pin_code>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<pin_code\>_ is device specific _setup PIN code_ determined in the step 4
+    and used to discover the device.
+
+##### Commissioning with long discriminator
+
+Use the following command pattern to discover devices with a long discriminator
+and try to pair with the first discovered one using the provided setup code.
+
+```
+$ chip-tool pairing onnetwork-long <node_id> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<pin_code\>_ and _<discriminator\>_ are device specific keys determined in
+    the step 4.
+
+##### Commissioning with QR code payload
+
+Matter devices log the QR code and its payload when they boot.
```

Comment:
Technically speaking, the log does not contain the QR code, but a proper link to a site with QR code and its payload. @Damian-Nordic, @doublemis1? BTW. Is it obligatory for all Matter devices? I guess no, since it would mandate the GUI for all accessories. Maybe better just write that the Matter device must somehow inform the user about the credentials, e.g. the QR code might be as well printed on the device's casing.

Reply:
You are right, the manufacturer needs somehow provide information or a way to obtain onboarding parameters:
generated QR code based on base-38 payload
emulated NFC with base-38 payload
generated manual pairing code
In our examples, we provide information through logs (it can be also achieved by NFC with proper config, but it doesn;t work with chip tool)